src/test/java/com/conveyal/datatools/manager/models/FeedVersionTest.java

```diff
@@ -106,4 +103,45 @@ void canDetectBlockingErrorTypesForPublishing(NewGTFSErrorType errorType) throws
 
         assertThat(feedVersion1.hasBlockingIssuesForPublishing(), equalTo(true));
     }
+
+    /**
+     * {@link FeedSource::publishedVersionId} should be unset if it references a feed version being deleted.
+     */
+    @Test
+    void shouldDeletePublishedVersionIdWhenDeletingVersion() {
+        final String NAMESPACE = "published_namespace";
+
+        FeedSource storedFeedSource = Persistence.feedSources.getOneFiltered(eq("projectId", project.id));
+        String feedSourceId = storedFeedSource.id;
+
+        // Create a feed version linked to the feed source.
+        FeedVersion feedVersion1 = new FeedVersion(feedSource);
+        feedVersion1.namespace = NAMESPACE;
+        feedVersion1.feedSourceId = feedSourceId;
+        Persistence.feedVersions.create(feedVersion1);
+
+        // Other a feed version linked to the feed source.
```

Comment:
Clarify this comment

Reply:
Thanks... fixed in https://github.com/ibi-group/datatools-server/pull/528/commits/bec4fa2b7e66cd721327c64b18c46d24a4a5742c.